corehq/apps/domain/decorators.py

```diff
@@ -90,10 +93,8 @@ def call_view(): return view_func(req, domain_name, *args, **kwargs)
                 return redirect_for_login_or_domain(req, login_url=login_url)
 
         couch_user = _ensure_request_couch_user(req)
-        if not domain_obj.is_active:
-            return _inactive_domain_response(req, domain_name)
         if domain_obj.is_snapshot:
-            if not hasattr(req, 'couch_user') or not req.couch_user.is_previewer():
+            if not req.couch_user.is_previewer():
```

Comment:
I like this suggestion. The falsy check was redundant (which I think you pointed out), because `ensure_request_couch_user` removes the possibility that code reaches here with a `None` user